server/ERNI.PBA.Server.Host/Controllers/RequestController.cs

```diff
@@ -31,6 +31,7 @@ public class RequestController : Controller
         private readonly IUnitOfWork _unitOfWork;
         private readonly MailService _mailService;
         private readonly ILogger _logger;
+        private readonly IConfiguration _configuration;
```

Comment:
you can remove this if its not used anymore

Reply:
unnecessary field has been removed